tests/test_lanes.py

```diff
@@ -84,11 +84,11 @@ def test_create_lanes_for_large_collection(self):
         for lane in lanes:
             eq_(self._default_library, lane.library)
             # They all are restricted to English and Spanish.
-            eq_(x.languages, languages)
+            eq_(lane.languages, languages)
```

Comment:
`x` was the wrong variable name. Python 3 caught this error because `x` goes out of scope once the list comprehension completes.